### PR TITLE
CSI-3720 downgrade controller builder base image

### PR DESCRIPTION
**Pull request by oriyarde**

### CODE REVIEW

File: Dockerfile-csi-controller.test [hunk 1]
```diff
@@ -16,7 +16,7 @@
 #    This Dockerfile.test is for running the csi controller local tests inside a container.
 #    Its similar to the Dockerfile, but with additional requirements-tests.txt and ENTRYPOINT to run the local tests.
 
-FROM registry.access.redhat.com/ubi8/python-38:1-77 as builder
+FROM registry.access.redhat.com/ubi8/python-38:1-75.1638364053 as builder
```
Comment (ArbelNathan) [inline Dockerfile-csi-controller.test:19]: no need for the 3 more places that the version mentioned?

Comment (oriyarde) [reply Dockerfile-csi-controller.test:19]: nope

Comment (ArbelNathan) [reply Dockerfile-csi-controller.test:19]: ok

**Review by ArbelNathan — APPROVED** [review]
LGTM

File: Dockerfile-csi-controller.test (hunk 1 above)
Comment (oriyarde) [reply Dockerfile-csi-controller.test:19]: you were right, need to to do it in the non-test dockerfile as well

**Pull request merged**